xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/companylogo.vm

```diff
@@ -46,7 +46,8 @@
   #end
 #end
 <div id="companylogo">
-  <a href="$!xwiki.getURL($services.wiki.currentWikiDescriptor.mainPageReference)" title="Home" rel="home" #if(!$displayPageHeader)class="navbar-brand"#end>
-    <img src="$!logourl" alt="Wiki Logo"/>
+  <a href="$!xwiki.getURL($services.wiki.currentWikiDescriptor.mainPageReference)" rel="home" #if(!$displayPageHeader)class="navbar-brand"#end>
```

Comment:
I don't think that removing the title from the link is good as it means that users won't get any hint anymore that the link points to the home page of the wiki when they're hovering the logo. I understand that the title attribute might not be enough for screen readers, but I think it would still be good to keep the title attribute, but maybe with the new translated text.

Reply:
My bad, I didn't realize this title was important for usability.
I added it back in https://github.com/xwiki/xwiki-platform/pull/3510/commits/a7411b82b82ef41d276bd817bac15aa5aa25d92e :+1: 

For AT, the title on a link is only used if there's no other text alternative, so having it is not an issue.